src/data/finset/noncomm_prod.lean

```diff
@@ -5,6 +5,8 @@ Authors: Yakov Pechersky
 -/
 
 import algebra.big_operators.basic
+import group_theory.submonoid.basic
+import group_theory.subgroup.basic
```

Comment:
These imports feel backwards to me - can we put these lemmas next to their commutative counterparts in `submonoid/membership` and wherever the subgroup lemmas are?

Reply:
I don't think it's all that more odd than `finset.prod`, so I think that location is fine.

Reply:
Moved, and adjusted to the naming conventions there (no `finset_` prefix in the lemma about `prod_mem` and `noncomm_prod_mem`; and also the order and names of parameters.